Web/CCMS/src/app/credit-card-add/credit-card-add.component.ts

```diff
@@ -108,10 +110,11 @@ export class CreditCardAddComponent {
   onSubmit() {
     console.log(this.creditCardForm.value);
     this.creditCardService
-
       .postCreditCard(this.formToCreditCard(this.creditCardForm))
       .subscribe();
+    this.location.back();
```

Comment:
jeg tænker det ville være fedt at den kun går tilbage hvis det lykkes at slette ;D vi bør vente på at kaldet er færdig før vi returnere

Reply:
kan man lave en .AtSuccess( {
this.location.back(); 
})